### PR TITLE
Expand contributing advice.

### PR DESCRIPTION
**Pull request by ericherman**

### CODE REVIEW

**Comment by ericherman** [comment]
This draft PR is (so far) only in english.

Once the "en" text is good enough, I would like some help with the "nl" version.

**Review by Ainali — APPROVED** [review]
That is a good and helpful addition!

File: src/pages/using-signalen.en.md [hunk 1]
```diff
@@ -25,7 +17,22 @@ Then you can contribute in the following ways:
 - Check outstanding feature requests on the [feature request backlog](https://github.com/orgs/Signalen/projects/2).
 - Make a new [feature request](https://github.com/Signalen/product-steering/issues/new?assignees=&labels=enhancement&template=feature_request.md&title=%5BFEATURE-REQUEST%5D) using the [feature request template](https://github.com/Signalen/product-steering/tree/main/.github/ISSUE_TEMPLATE) if your requirement or wish is not listed. Note: Please fill it in completely otherwise your feature request cannot be processed.
 - Announce your feature request on the [Signalen community mailinglist](https://lists.publiccode.net/mailman/postorius/lists/signalen-discuss.lists.publiccode.net/)
+- If you have developers intending to contribute to Signalen, ensure that at least one of them attends the [Weekly Technical Steering call](https://signalen.org/en/contact/) to inform scoping of the work and the technical soundness of possible solutions.
```
Comment (tcoenen) [inline src/pages/using-signalen.en.md:20]: good to add this!

File: src/pages/using-signalen.en.md [hunk 2]
```diff
@@ -25,7 +17,22 @@ Then you can contribute in the following ways:
...
 
 Would you like to participate in the product steering group? Or would you like to know more about how decisions are made in the Signalen community, please [contact](/en/contact/) us.
+
+## For developers
+
+Are you a developer and would like to help us with the further development of Signalen? Please contact us via the [Common Ground Slack](https://join.slack.com/t/samenorganiseren/shared_invite/zt-dex1d7sk-wy11sKYWCF0qQYjJHSMW5Q) to explore how we can make Signalen better together.
+
+- Check out the [Signalen-repository on Github.](https://github.com/signalen)
+- [User Guide and Implementation Guide](https://docs.signalen.org)
+- Check out the [Signalen API documentation](https://api.data.amsterdam.nl/api/swagger/?url=/signals/swagger/openapi.yaml#/default/get_signals_v1_private_signals_geography)
```
Comment (tcoenen) [inline src/pages/using-signalen.en.md:32]: Nit: remove the path to the signals geography endpoint, just open the documentation page at the top. (So, remove: `#/default/get_signals_v1_private_signals_geography`.)

**Review by tcoenen — CHANGES_REQUESTED** [review]
Nothing controversial here, the emphasis on joining the weekly meetings is much appreciated.

I have one small comment.

File: src/pages/using-signalen.en.md (hunk 2 above)
Comment (ericherman) [reply src/pages/using-signalen.en.md:32]: Good catch! That wasn't added, rather it was moved as-is from above. Happy to fix that.

**Review by tcoenen — APPROVED** [review]
Akkoord!

**Review by felixfaassen — APPROVED** [review]
Great addition. I will make the nl version once this is merged

**Review by Ainali — APPROVED** [review]

**Comment by ericherman** [comment]
> Great addition. I will make the nl version once this is merged

Okay, yes, we can do .nl as a separate PR.

**Pull request merged**